flink-kubernetes-webhook/pom.xml

```diff
@@ -24,7 +24,7 @@ under the License.
         <groupId>org.apache.flink</groupId>
         <artifactId>flink-kubernetes-operator-parent</artifactId>
         <version>1.11-SNAPSHOT</version>
-        <relativePath>..</relativePath>
```

Comment:
is this necessary? should it be consistent with the other modules?

Reply:
IntelliJ was complaining that `..` was invalid and as I was moving the profile up to the parent it felt important to fix the warning. I haven't checked if the intellij error is correct or if it should be applied else where.

Reply:
So checking https://maven.apache.org/ref/3.9.9/maven-model/maven.html#parent intellij is correct `..` is not a valid reference, but based on stackoverflow and some other sources it might have been at one point or at least commonly thought to be.